tests/benchdnn/dnnl_memory.cpp

```diff
@@ -58,6 +58,20 @@ dnn_mem_t::dnn_mem_t(const_dnnl_memory_desc_t md, dnnl_engine_t engine,
     }
 }
 
+// todo: maybe align better with other ctors
+dnn_mem_t::dnn_mem_t(const_dnnl_memory_desc_t md, void *value) {
+    auto status = dnnl_memory_desc_clone(&md_, md);
```

Comment:
Might worth checking that md is for host_scalar and return if it's not before initializing `md_`.

Reply:
Done!